_scripts/release_component.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+NEW_RELEASE="${NEW_RELEASE:-unknown}"
+DRY_RUN_CMD='echo'
+DRY_RUN=YES
+UPDATE_MASTER=YES
+
+# flags are --help --no-dry-run --no-update-master
+for i in "$@"; do
```

Comment:
I don't think you actually want to quote the `$@`... if you pass in multiple flags you'll probably see wonky behavior... (it'll treat all items in `$@` as a single arg)